Tasks/DockerV1/make.json

```diff
@@ -5,6 +5,12 @@
                 "node_modules/azure-pipelines-tasks-docker-common/node_modules/azure-pipelines-task-lib"
             ],
             "options": "-Rf"
+        },
+        {
+            "items": [
+                "node_modules/azure-pipelines-tasks-azure-arm-rest-v2/node_modules/azure-pipelines-task-lib"
```

Comment:
You can put this into array of previous step

Reply:
Fixed